src/main/java/org/jitsi/jicofo/JitsiMeetConferenceImpl.java

```diff
@@ -2154,6 +2170,69 @@ public MuteResult handleMuteRequest(Jid muterJid, Jid toBeMutedJid, boolean doMu
         return succeeded ? MuteResult.SUCCESS : MuteResult.ERROR;
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    public void muteAllNonModeratorParticipants(MediaType mediaType)
+    {
+        for (Participant participant : participants)
```

Comment:
I think we need the lock on `participantsLock` here, but we should not hold it while waiting for a response from the bridge.

Reply:
It is done in muteParticipant. You want me to rename those?